modules/gitops-enabling-the-creation-of-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,270 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-aggregated-cluster-roles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-creation-of-aggregated-cluster-roles_{context}"]
+= Enabling the creation of aggregated cluster roles
+
+To enable the creation of aggregated cluster roles for the Argo CD Application Controller component of a cluster-scoped Argo CD instance, you must configure the corresponding field by editing the YAML file of the Argo CD custom resource (CR).
+
```

Comment:
missing prereqs

Reply:
Thanks a lot for the feedback. Please refer to the Assembly prerequisites section [here](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines). The GitOps standalone documentation follows this method. To keep it consistent with what we already have, I prefer to retain this section as is.